src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -75,13 +76,19 @@ CPU::CPU() {
             }
         }
         _processors = processors.size();
-        _numa_nodes = sockets.size();
+        _numa_nodes = sockets.size() == 0 ? 1 : sockets.size();
         for (auto&& socket : sockets) {
             _cores += socket.second;
         }
         if (_cores == 0) {
             _cores = _processors;
         }
+        if (_processors > 0 && _numa_nodes > 0 && _cores > 0) {
+            get_cpu_mapping_from_cores(_processors, _numa_nodes, _cores, _proc_type_table, _cpu_mapping_table);
+        } else {
+            IE_THROW() << "Wrong CPU information. processors: " << _processors << ", numa_nodes: " << _numa_nodes
```

Comment:
Please use `OPENVINO_THROW` to replace `IE_THROW`.

Reply:
Ok, done.